Explorer/Assets/DCL/Profiles/RealmProfileRepository.cs

```diff
@@ -0,0 +1,68 @@
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.WebRequests;
+using ECS;
+using Ipfs;
+using System.Threading;
+
+namespace DCL.Profiles
+{
+    public class RealmProfileRepository : IProfileRepository
+    {
+        private readonly IWebRequestController webRequestController;
+        private readonly IRealmData realm;
+        private readonly CacheProfileRepository cacheProfileRepository;
+        private readonly URLBuilder urlBuilder = new ();
+
+        public RealmProfileRepository(IWebRequestController webRequestController,
+            IRealmData realm,
+            CacheProfileRepository cacheProfileRepository)
```

Comment:
`Cache` should be under its own abstract contract that is suitable for cache only